src/shared/lib/oql/test-oql-parser.js

```diff
@@ -96,13 +56,164 @@ doTest("TP53:PROT<=-2\n", [{gene:"TP53", alterations:[{alteration_type: "prot",
 
 doTest("BRAF:MUT=V600E", [{gene:"BRAF", alterations:[{alteration_type: "mut", constr_rel: "=", constr_type:"name", constr_val:"V600E", info:{}}]}])
 doTest("BRAF:MUT=V600", [{gene:"BRAF", alterations:[{alteration_type: "mut", constr_rel: "=", constr_type:"position", constr_val:600, info:{amino_acid:"V"}}]}])
-doTest("BRAF:FUSION MUT=V600", [{gene:"BRAF", alterations:[{alteration_type:'fusion'}, {alteration_type: "mut", constr_rel: "=", constr_type:"position", constr_val:600, info:{}}]}])
+doTest("BRAF:FUSION MUT=V600", [{gene:"BRAF", alterations:[{alteration_type:'fusion'}, {alteration_type: "mut", constr_rel: "=", constr_type:"position", constr_val:600, info:{"amino_acid":"V"}}]}])
 doTest("BRAF:FUSION", [{gene:"BRAF", alterations:[{alteration_type:'fusion'}]}])
 doTest("MIR-493*:MUT=V600", [{gene:"MIR-493*", alterations:[{alteration_type: "mut", constr_rel: "=", constr_type:"position", constr_val:600, info:{amino_acid:"V"}}]}])
 
 doTest("BRAF:CNA >= gain", [{gene:"BRAF", alterations:[{alteration_type:"cna", constr_rel:">=", constr_val:"GAIN"}]}])
 doTest("BRAF:CNA < homdel", [{gene:"BRAF", alterations:[{alteration_type:"cna", constr_rel:"<", constr_val:"HOMDEL"}]}])
 
+doTest("[TP53 BRCA1] NRAS", 
```

Comment:
in the future we should bring all these over to the new code, but for now just please put your tests into typescript (.spec.ts), which should be possible since it looks like you've got es6 imports working from the parser. Because these tests are legacy and dont run automatically as part of our test system.

Reply:
@adamabeshouse can you help us with an example? Then perhaps @sandertan can rewrite his test. Just want to check we understand your suggestion and make sure we don't start some large refactoring effort of the tests here, which is not the goal of our PR.

Reply:
forget about it, I'll just do it later all at once